distribution/packages/build.gradle

```diff
@@ -167,6 +167,10 @@ Closure commonPackageConfig(String type, boolean oss) {
     configurationFile '/etc/elasticsearch/elasticsearch.yml'
     configurationFile '/etc/elasticsearch/jvm.options'
     configurationFile '/etc/elasticsearch/log4j2.properties'
+    configurationFile '/etc/elasticsearch/role_mapping.yml'
```

Comment:
Shouldn't this be conditional on the `oss` flag?

Reply:
+1, these should be conditionally added only.